integration-libs/opf/base/root/services/opf-resource-loader.service.ts

```diff
@@ -74,6 +71,30 @@ export class OpfResourceLoaderService {
     return this.scriptLoader.hasScript(src);
   }
 
+  /**
+   * Create attributes intended to script and link elements.
+   *
+   * Return attributes list including keyValueList and OPF specific attribute with below logic:
+   *
+   * 1. Resource loads only once: 'opf-load-once' key detected, no additional attribute added.
+   * 2. Resource deleted at page/payment change: 'data-opf-resource' attribute is added.
+   */
+
+  protected createAttributesList(keyValueList?: OpfKeyValueMap[] | undefined): {
+    [key: string]: string;
+  } {
+    const attributes: { [key: string]: string } = {};
+    keyValueList?.forEach((keyValue: OpfKeyValueMap) => {
+      attributes[keyValue.key] = keyValue.value;
+    });
+    if (!attributes[this.OPF_RESOURCE_LOAD_ONCE_ATTR_KEY]) {
```

Comment:
Just realized, we only check its presence but not value 'true'.
Still valid as server as unique value ('true') associated with this key.
It could be more robust by checking it.